com.github.Nheko.json

```diff
@@ -0,0 +1,210 @@
+    "--socket=wayland",
+    "--socket=x11",
+    "--talk-name=org.freedesktop.Notifications"
+  ],
+  "id": "com.github.Nheko",
+  "modules": [
+    {
+      "build-commands": [
+        "make install"
+      ],
+      "buildsystem": "simple",
+      "name": "lmdb",
+      "sources": [
+        {
+          "sha256": "f3927859882eb608868c8c31586bb7eb84562a40a6bf5cc3e13b6b564641ea28",
+          "type": "archive",
+          "url": "https://github.com/LMDB/lmdb/archive/LMDB_0.9.22.tar.gz"
+        }
+      ],
+      "subdir": "libraries/liblmdb"
+    },
+    {
+      "build-commands": [
+        "cmake -H. -Bbuild -DCMAKE_BUILD_TYPE=Release -DSPDLOG_BUILD_EXAMPLES=0 -DSPDLOG_BUILD_TESTING=0",
+        "cmake --build build --target install"
+      ],
+      "buildsystem": "simple",
+      "name": "spdlog",
+      "sources": [
+        {
+          "sha256": "94f74fd1b3344733d1db3de2ec22e6cbeb769f93a8baa0d4a22b1f62dc7369f8",
+          "type": "archive",
+          "url": "https://github.com/gabime/spdlog/archive/v0.17.0.tar.gz"
+        }
+      ]
+    },
+    {
+      "build-commands": [
+        "cmake -H. -Bbuild -DCMAKE_BUILD_TYPE=Release",
+        "cmake --build build --target install"
+      ],
+      "buildsystem": "simple",
+      "name": "matrix-structs",
+      "sources": [
+        {
+          "sha256": "255a82a9e463261858cd7da18f34fd60d19882ab507d214e47ea47ff78ae68b6",
+          "type": "archive",
+          "url": "https://github.com/mujx/matrix-structs/archive/8de04afea34e95c14d1dde82af390592dfde90dd.tar.gz"
+        },
+        {
+          "dest": "include",
+          "sha256": "fbdfec4b4cf63b3b565d09f87e6c3c183bdd45c5be1864d3fcb338f6f02c1733",
+          "type": "file",
+          "url": "https://github.com/nlohmann/json/releases/download/v3.1.2/json.hpp"
+        },
+        {
+          "dest": "include",
+          "sha256": "d815424af42f2fb26994bc2e58c836a2e8fb7ea635d1b23cc6d6235a7af851ee",
+          "type": "file",
+          "url": "https://github.com/mpark/variant/releases/download/v1.3.0/variant.hpp"
+        }
+      ]
+    },
+    {
+      "build-commands": [
+        "cmake -H. -Bbuild -DBUILD_SHARED_LIBS=ON -DCMAKE_BUILD_TYPE=Release",
+        "cmake --build build --target install --config Release"
+      ],
+      "buildsystem": "simple",
+      "name": "olm",
+      "sources": [
+        {
+          "commit": "730e5e719c6c6ace4e1862ceacf0bb4f7c626251",
+          "disable-shallow-clone": true,
+          "tag": "2.2.2",
+          "type": "git",
+          "url": "https://git.matrix.org/git/olm.git"
+        },
+        {
+          "dest": ".",
+          "path": "cmake/CMakeLists.txt",
+          "type": "file"
+        },
+        {
+          "dest": "cmake",
+          "path": "cmake/OlmConfig.cmake.in",
+          "type": "file"
+        }
+      ]
+    },
+    {
+      "build-commands": [
+        "./configure --prefix=/usr",
+        "make && make check",
+        "make install"
```

Comment:
The default `buildsystem` is `autotools`, so this can just be removed entirely.

Reply:
Are there any docs with the default commands for each `buildsystem` and how to overwrite them?